src/SprykerEco/Service/ArvatoRss/Iso3166Converter.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Service\ArvatoRss;
+
+use Spryker\Service\Kernel\AbstractService;
+
+class Iso3166Converter extends AbstractService implements Iso3166ConverterInterface
+{
+
+    /**
+     * @const string ISO2_KEY
```

Comment:
no need to repeat the key here (ISO2_KEY)

Reply:
Removed comments from constants here.